src/guide/frontend/aspnet.md

```diff
@@ -164,72 +172,79 @@ With the "Passwordless ASP.NET Identity SDK," you can streamline this process by
 
 Upon a successful authentication, our sample application will automatically redirect you to the /Authorized/HelloWorld page, which requires you to be logged in to access.
 
-```html
-@page @model LoginModel @using Microsoft.Extensions.Options @using Passwordless.AspNetCore @inject
-IOptions<PasswordlessAspNetCoreOptions>
-  PasswordlessOptions; @{ ViewData["Title"] = "Login"; }
-  <h1>@ViewData["Title"]</h1>
+```
```

Comment:
Same here

Reply:
So if you add html, the linter will add an unnecessary tag and break the code block